### PR TITLE
Framed fix

### LINKED ISSUE #2134: nested fboxes are collapsed to a single one
When `\fbox` is nested, e.g. to create a double border, the current master flattens it to a single box, thus a single border.
```latex
\documentclass{article}
\begin{document}
\fbox{\fbox{nested box}}
\end{document}
```
PDF 
image
XML
```xml
<para xml:id="p1">
  <p><text framecolor="black" framed="rectangle">nested box</text></p>
</para>
```

### LINKED ISSUE #2135: value of \fboxsep is not preserved
When outputting a `\fbox`, the value of `\fboxsep` is ignored. The end result is that every framed box has no padding at all in the default stylesheet – at a minimum, the default CSS could include the usual 3pt padding. Minimal example is
```latex
\documentclass{article}
\begin{document}
\fbox{box}
\end{document}
```
XML output
```xml
<para xml:id="p1">
  <p><text framecolor="black" framed="rectangle">box</text></p>
</para>
```
even though the value of `\fboxsep` is 3pt (set within LaTeX.pool.ltxml). Changing `\fboxsep` has no effect, it seems like the value just is not used.

### PR DESCRIPTION
**Pull request by brucemiller**
This PR uses the `framed` attribute, vs `ltx_framed` CSS a bit more consistently, allows for adjusting the frame separation, and avoids collapsing nodes which have framing.

Fixes #2134 
Fixes #2135

### CODE REVIEW

File: lib/LaTeXML/Package/LaTeX.pool.ltxml [hunk 1]
```diff
@@ -4695,7 +4695,7 @@ DefMacro('\framebox', '\@ifnextchar(\pic@framebox\@framebox', robust => 1);
 DefConstructor('\@framebox[Dimension][]{}',
   "?#mathframe(<ltx:XMArg enclose='box'>#inner</ltx:XMArg>)"
     . "(<ltx:text ?#width(width='#width') ?#align(align='#align')"
-    . " framed='rectangle' framecolor='#framecolor'"
+    . " framed='rectangle' framecolor='#framecolor' cssstyle='#cssstyle'"
```
Comment (dginev) [inline lib/LaTeXML/Package/LaTeX.pool.ltxml:4698]: I am spiritually obliged to mention `css_style` looks better on every PR that uses the non-underscore variant :>

File: t/graphics/xcolors.xml [hunk 2]
```diff
@@ -332,307 +332,307 @@ A=100*13 = 1300pt;</p>
...
-            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 0 1 1 0</text></td>
-            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 0 1 1</text></td>
-            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> FF0000</text></td>
-            <td align="left"><text backgroundcolor="#4D4D4D" class="ltx_phantom" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 0.3</text></td>
+            <td align="left"><text backgroundcolor="#FF0000" class="ltx_phantom" cssstyle="padding:0.0pt" font="sansserif" fontsize="90%" framecolor="black" framed="rectangle">XX</text><text font="sansserif" fontsize="90%"> 1 0 0</text></td>
```
Comment (dginev) [inline t/graphics/xcolors.xml:335]: Quickly marking that the test using `ltx_phantom` text now gets explicit 0 padding. 

Probably intentional?

**Review by dginev — APPROVED** [review]
Looks healthy! I left a couple of minor comments, but nothing significant.

File: lib/LaTeXML/Package/LaTeX.pool.ltxml (hunk 1 above)
Comment (brucemiller) [reply lib/LaTeXML/Package/LaTeX.pool.ltxml:4698]: `css_style`??? *so* boring...

File: t/graphics/xcolors.xml (hunk 2 above)
Comment (brucemiller) [reply t/graphics/xcolors.xml:335]: I think so.

**Pull request merged**